app/uk/gov/hmrc/tradergoodsprofilesdatastore/connectors/RouterConnector.scala

```diff
@@ -50,12 +54,9 @@ class RouterConnector @Inject() (config: Configuration, httpClient: HttpClientV2
       .map(_ => Done)
 
   def getRecords(
-    eori: String,
-    lastUpdatedDate: Option[String] = None,
-    page: Option[Int] = None,
-    size: Option[Int] = None
+    eori: String
   )(implicit hc: HeaderCarrier): Future[HttpResponse] =
```

Comment:
I just realised what you had here before makes sense since we only have one request for get, sorry!

Reply:
reverted!